Bottom: 42c07b00640f3d7eac1bec28680111bdd911d89a
Top:    e4b970cda4d669fb32c1691a9e5dd0699369cf9d
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-24 16:25:42 +0800

Refresh of autofs-5.0.7-use-LIBS-for-link-libraries.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index a0af868..897a2b4 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -63,6 +63,7 @@
 - add missing libtirpc lib to mount_nfs.so when TIRPC enabled.
 - use compiler determined by configure instead of hard-coded ones.
 - remove hard-coded STRIP variable.
+- use LIBS for link libraries.
 
 25/07/2012 autofs-5.0.7
 =======================
